docs/parallel/concrt/reference/invalid-scheduler-policy-thread-specification-class.md

```diff
@@ -22,7 +21,7 @@ class invalid_scheduler_policy_thread_specification : public std::exception;
 
 |Name|Description|
 |----------|-----------------|
-|[invalid_scheduler_policy_thread_specification](invalid-scheduler-policy-value-class.md#ctor|Overloaded. Constructs an `invalid_scheduler_policy_value` object.|
+|[invalid_scheduler_policy_thread_specification](#ctor)|Overloaded. Constructs an `invalid_scheduler_policy_thread_specification` object.|
```

Comment:
Drive-by fix where `invalid_scheduler_policy_thread_specification` and `invalid_scheduler_policy_value` were mixed up.